<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotationWidget</class>
 <widget class="QWidget" name="RotationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(160, 160, 160);</string>
  </property>
  <widget class="QComboBox" name="axis_combobox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="angle_spinbox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="unit_combobox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
